app/models/vendor_api_request.rb

```diff
@@ -1,3 +1,32 @@
 class VendorAPIRequest < ApplicationRecord
   belongs_to :provider, optional: true
+  scope :validation_errors, -> { where(status_code: 422) }
```

Comment:
This implies that we are only getting back the errors so consider renaming, maybe `validation_error_requests`?

Reply:
Potentially `with_validation_errors` since it's a scope on `requests` already?

Reply:
I did initially think of that, but I wonder if it may cause some confusion when used on line 6. 

E.g. 
```ruby
distinct_errors = with_validation_errors.flat_map do |request|
```

Reply:
I went through a few iterations of this but landed on `unprocessable_entities` as felt like the most suitable and maps to the response we return to users